broker/cloud_functions/filter_exgalac_trans/main.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python3
+# -*- coding: UTF-8 -*-
+
+"""Filter alerts for likely extragalactic transients."""
+
+import astropy.units as u
+import base64
+from google.cloud import logging
+import numpy as np
+import os
+
+from broker_utils import data_utils, gcp_utils, schema_maps
+
+
+PROJECT_ID = os.getenv("GCP_PROJECT")
+TESTID = os.getenv("TESTID")
+SURVEY = os.getenv("SURVEY")
+
+# connect to the logger
+logging_client = logging.Client()
+log_name = "filter-exgalac-trans-cloudfnc"  # same log for all broker instances
+logger = logging_client.logger(log_name)
+
+# GCP resources used in this module
+ps_topic = f"{SURVEY}-exgalac_trans_cf"
+if TESTID != "False":  # attach the testid to the names
+    ps_topic = f"{ps_topic}-{TESTID}"
+
+schema_map = schema_maps.load_schema_map(SURVEY, TESTID)
+
+
+def run(msg: dict, context) -> None:
```

Comment:
Is the deal here that `run` must accept an argument `context`?

Reply:
Yes that's right. This function is triggered by a Pub/Sub message via Cloud Functions, which expects both arguments. I added documentation.